pkg/daemon/daemon.go

```diff
@@ -291,6 +315,76 @@ func (dn *Daemon) getHealth() error {
 	return nil
 }
 
+// stateAndConfigs is the "state" node annotation plus parsed machine configs
+// referenced by the currentConfig and desiredConfig annotations.
+type stateAndConfigs struct {
```

Comment:
This struct and fields are never mean to be accessed outside of the `daemon` package, correct?

Reply:
Yep!